client/src/features/Home/components/MessageCard.tsx

```
@@ -13,23 +13,22 @@ type MessageCardProps = {
 
 export const MessageCard = ({ message, image, index }: MessageCardProps) => {
   return (
-    <Flex
+    <Card
       role="article"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<article>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)